framework/service/org/moqui/impl/InstanceServices.xml

```diff
@@ -348,6 +359,42 @@ along with this software (see the LICENSE.md file). If not, see
             ec.logger.info("Initializing docker container for ${appInstance.instanceName} with:\n${jsonBody}")
             restClient.jsonObject(jsonBody)
 
+            try {
+                authToken = instanceImage.password
+                if (((String) instanceImage.username).equals("AWS")) {
```

Comment:
@rohitpawar2811 
Instead of having check on userName, we can extend InstanceImage entity to store the processCommand

Reply:
Yes @dixitdeepak you are right I think we can go on this way : 
we can create a field named `authTokenCmd`  into `moqui.server.instance.InstanceImage` and then we can check based on that if authTokenCmd is not empty then go for that , otherwise use password for authentication, From this way we can make it more generic and other registries like aws (which generate token diffrent by using cmd) can easily configured.